psychopy/sound/__init__.py

```diff
@@ -198,8 +198,18 @@
 if Sound is not None:
     audioLib = thisLibName
     init = backend.init
-    if hasattr(backend, 'getDevices'):
+    getDevices = None
+    if audioLib != 'ptb' and hasattr(backend, 'getDevices'):
```

Comment:
Isn't this already handled by `SpeakerDevice.getAvailableDevices`? As that imports `sound.getDevices` which then handles the different backends